azext_hanaonazure/_params.py

```diff
@@ -11,4 +11,11 @@
 def load_arguments(self, _):
     with self.argument_context('hanainstance') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
-        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
\ No newline at end of file
+        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
+        c.argument('hana_vnet', options_list=['--hana-vnet', '-hv'], help="ARM ID of an Azure Vnet with access to the HANA instance.")
```

Comment:
Do you need `id_part='` for all of these?

Reply:
I believe that only pieces of the ARM id have that part